HBASE.md

```diff
@@ -66,14 +66,33 @@ put 'sbr_control_db:enterprise_201801', '3210000001', 'd:grp_turnover', '101'
 put 'sbr_control_db:enterprise_201801', '3210000001', 'd:app_turnover', '102'
 put 'sbr_control_db:enterprise_201801', '3210000001', 'd:ent_turnover', '103'
 put 'sbr_control_db:enterprise_201801', '3210000001', 'd:prn', '0.038709584'
+put 'sbr_control_db:enterprise_201801', '3210000001', 'd:working_props', '183'
+put 'sbr_control_db:enterprise_201801', '3210000001', 'd:employment', '34'
+put 'sbr_control_db:enterprise_201801', '3210000001', 'd:region', 'region-1'
 
 put 'sbr_control_db:enterprise_201802', '3210000001', 'd:ern', '1000000123'
-put 'sbr_control_db:enterprise_201802', '3210000001', 'd:name', '201802-name-1000000123'
-put 'sbr_control_db:enterprise_201802', '3210000001', 'd:address1', '201802-address1-1000000123'
-put 'sbr_control_db:enterprise_201802', '3210000001', 'd:postcode', '201802-postcode-1000000123'
-put 'sbr_control_db:enterprise_201802', '3210000001', 'd:sic07', '201802-sic07-1000000123'
-put 'sbr_control_db:enterprise_201802', '3210000001', 'd:legal_status', '201802-legalstatus-1000000123'
-put 'sbr_control_db:enterprise_201802', '3210000001', 'd:prn', '0.049810695'
+put 'sbr_control_db:enterprise_201802', '3210000001', 'd:entref', '201801-entref-1000000123'
```

Comment:
You probably want to prefix the values of these variables with the correct period.

This was just done to make it easy to scan the results to ensure that the correct table was hit.